db/routines.sql

```diff
@@ -0,0 +1,50 @@
+CREATE OR REPLACE FUNCTION lock_preset(IN preset_name CHARACTER)
+	RETURNS BOOLEAN AS
```

Comment:
this file contains both, tabs and spaces used for indentation, please keep it consistent with rest of the files (I believe we use spaces)

Reply:
fixed